packages/core/src/Ogmios/OgmiosToCardanoWasm.ts

```diff
@@ -1,40 +1,34 @@
-import CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
+import { CSL } from '@cardano-sdk/cardano-serialization-lib';
 import OgmiosSchema from '@cardano-ogmios/schema';
 import * as Asset from '../Asset';
 
 export const OgmiosToCardanoWasm = {
```

Comment:
Should inject `CardanoSerializationLib` because we want to use the same wasm object everywhere. Not sure if that's in scope, but if it is might as well rename this to `OgmiosToCSL` at the same time because the interface will change

Reply:
Yes, I also just picked this up. Will fix and rename

Reply:
Done in e9a242170cdca968c39f281b254edd2bef297b65